src/views/admin/PendingAccounts.vue

```diff
@@ -161,12 +176,42 @@
           })
         )
       },
+      async discardUserAccount(account: any): Promise<void> {
+        this.selectedItem = account
+        this.isWaitingForValidation = true
+        try {
+          await account.discardCreateRequest()
+        } catch (err) {
+          this.isWaitingForValidation = false
+          if ((err as Error).message === "User canceled the dialog box") {
+            return
+          }
+
+          this.$msg.error(
+            this.$gettext(
+              "An unexpected issue occurred while approving " +
+                "the wallet account creation of user %{ name }",
+              {
+                name: account.name,
+              }
+            )
+          )
+          throw err
```

Comment:
are we already doing this all time ? We may need to discuss about this to be consistent. For me a console.log is okay, but throwing the error might trigger some unwanted error handling code. Furthermore, it seems here what you want is to throw an UIError